docs/resources/Poll.md

```diff
@@ -0,0 +1,138 @@
+# Poll Resource
+
+A poll is... well... a poll! It holds information about a poll!
+
+![Example message containing a poll](example-poll.png)
+
+### Poll Object
+
+The poll object has a lot of levels and nested structures. It was also designed
+to support future extensibility, so some fields may appear to be more complex than
+necessary.
+
+###### Poll Object Structure
+
+| Field             | Type                                                                                                | Description                                                     |
+|-------------------|-----------------------------------------------------------------------------------------------------|-----------------------------------------------------------------|
+| question          | [Poll Media Object](#DOCS_RESOURCES_POLL/poll-media-object-poll-media-object-structure)             | The question of the poll. Only `text` is supported.             |
+| answers           | List of [Poll Answer Objects](#DOCS_RESOURCES_POLL/poll-answer-object-poll-answer-object-structure) | Each of the answers available in the poll.                      |
+| expiry            | IS08601 timestamp                                                                                   | The time when the poll ends.                                    |
+| allow_multiselect | boolean                                                                                             | Whether a user can select multiple answers                      |
+| layout_type       | integer                                                                                             | The [layout type](#DOCS_RESOURCES_POLL/layout-type) of the poll |
+| results           | [Poll Results Object](#DOCS_RESOURCES_POLL/poll-results-object-poll-results-object-structure)       | The results of the poll                                         |
```

Comment:
This seems to be nullable for messages received over a gateway connection, I've yet to see it null over rest

Reply:
Our intention is that it should be always non-null (with just... default values for polls without votes yet). Are you still seeing it null through the Gateway? I'm going to check later just wondering if anyone knows right now. :P

Reply:
![image](https://github.com/discord/discord-api-docs/assets/61027276/4721765e-0c6b-49f8-b71b-a3981cc4fe6b)
I can still reproduce it